sdk/src/openssl/temp_signer.rs

```diff
@@ -287,129 +265,3 @@ pub fn get_temp_signer_by_alg<P: AsRef<Path>>(
         }
     }
 }
-
-fn make_key_path_pair<P: AsRef<Path>>(path: P, key_name: &str) -> (PathBuf, PathBuf) {
-    let mut sign_cert_path = path.as_ref().to_path_buf();
-    sign_cert_path.push(key_name);
-    sign_cert_path.set_extension("pub");
-    //println!("sign_cert_path = {:#?}", sign_cert_path);
-
-    let mut pem_key_path = sign_cert_path.clone();
-    pem_key_path.set_extension("pem");
-    //println!("pem_key_path = {:#?}", pem_key_path);
-
-    (sign_cert_path, pem_key_path)
-}
-
-// The .x509 directory at the root of this repo is flagged
```

Comment:
If this is no longer true, can we remove the .x509 exclusion from .gitignore?

Reply:
I think so.